src/nrnpython/inithoc.cpp

```diff
@@ -207,6 +207,33 @@ static int have_opt(const char* arg) {
     return 0;
 }
 
+#if __linux__ || DARWIN
```

Comment:
Should this be an `if` or an `ifdef`? Won't this cause a compilation error with an `if` if `__linux__` is not a defined token already (it apparently works with Clang on MacOS, which is admittedly somewhat strange)? We also use `ifdef` in other instances, for instance here:
https://github.com/neuronsimulator/nrn/blob/925a22567e4545ddeca1d9d4e97268d82e826aeb/src/nrnpython/nrnpython.cpp#L30

so I'd suggest using `#if defined(__linux__) || DARWIN` everywhere (we define `DARWIN` in the CMake config so that part is fine).

Reply:
I agree. Will change it.